js-escapes/index.html

```diff
@@ -9,6 +9,7 @@
 <noscript><strong>To use this tool, please <a href=http://enable-javascript.com/>enable JavaScript</a> and reload the page.</strong></noscript>
 <textarea autofocus>Fingerspitzengefühl is a German term.\nIt’s pronounced as follows: [ˈfɪŋɐˌʃpɪtsənɡəˌfyːl]</textarea>
 <div><label><input type=checkbox checked> only escape non-ASCII and unprintable ASCII characters</label></div>
+<div><label><input type=checkbox checked> use shorthand characters (\xB0 instead of \u00B0)</label></div>
```

Comment:
That makes sense. Then I don't have to store the negated value in storage and the default would be unchecked.